enterprise/mqtt/build.gradle

```diff
@@ -27,3 +32,12 @@ task copyResources {
     }
 }
 jar.dependsOn('copyResources')
+
+task jarTest (type: Jar) {
+    from sourceSets.test.output
+    classifier = 'mqtt-testing'
+}
```

Comment:
What does this task do? Is this only for generating a test jar?

Reply:
Yes